src/__tests__/index.test.ts

```diff
@@ -107,7 +107,7 @@ const samlConfig: SamlConfig = {
   // decryptionPvk: fs.readFileSync("./certs/key.pem", "utf-8"),
   identifierFormat: "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
   issuer: "https://spid.agid.gov.it/cd",
-  logoutCallbackUrl: "http://localhost:3000/slo",
+  // logoutCallbackUrl: "http://localhost:3000/slo",
```

Comment:
Is this on purpose?

Reply:
It is a cleanup action: I noticed that variable *logoutCallbackUrl* is never used in the software and was commented so I thought that needed to be removed.

Reply:
I think this is used in passport-saml. I'd leave this in for now.